js.js

```diff
@@ -0,0 +1,9 @@
+let checkDevice = navigator.userAgent;
+let training = document.querySelector('#training');
+console.log(checkDevice);
+
+if (navigator.userAgent.match('iPhone') || navigator.userAgent.match('Android') || 
+    navigator.userAgent.match('iPad') || navigator.userAgent.match('RIM')) {
```

Comment:
`navigator.userAgent.match('RIM')` - what is RIM ? xD

This solution isn't very good, because it works only for `iPhone`, `iPad`, `Android`, and `RIM`.

So we check by brand / OS... It's just not the best solution :)

Let's Google for a better one:

![image](https://user-images.githubusercontent.com/6686679/215525013-9792e9ed-d5ae-4817-a6ba-90e1c34ed918.png)

First URL on Google...

![image](https://user-images.githubusercontent.com/6686679/215525202-997f2aa5-a46f-4f06-b245-a22724cef591.png)

First answer in that question...

https://stackoverflow.com/a/4819886/1657101

```
function isTouchDevice() {
  return (('ontouchstart' in window) ||
     (navigator.maxTouchPoints > 0) ||
     (navigator.msMaxTouchPoints > 0));
}

if (isTouchDevice()) {
    document.body.classList.add('_touch');
    training.style.display = 'none';
}
```

This solution will also work for Motorola, Nokia, Xiaomi, Huawei, etc xD

Any browser that supports touch events :)
So any browser that is used on a touch device.

And also since it's the most popular solution on the Internet, it's most probably the best one :)

![image](https://user-images.githubusercontent.com/6686679/215526141-143e2d46-b7a7-4802-80d9-6e758a2a0d88.png)